dts/bindings/serial/silabs,gecko-usart.yaml

```diff
@@ -38,3 +38,14 @@ properties:
       type: array
       required: false
       description: CTS pin configuration defined as <location port pin>
+
+    pinctrl-0:
+      type: phandles
+      required: false
+      description: |
+        GPIO pin configuration for serial signals (RX, TX, RTS, CTS).  We expect
+        that the phandles will reference pinctrl nodes.
+
+        For example the USART1 would be
+           pinctrl-0 = <&usart1_tx_pb6 &usart1_rx_pb7>;
```

Comment:
this is redundant property is already defined in `pinctrl-device`, if you want to document something about UART please just use the top-level description.

Reply:
removed